packages/json_dynamic_widget/lib/src/components/processors/map_arg_processor.dart

```diff
@@ -24,18 +24,18 @@ class MapArgProcessor implements ArgProcessor {
   ];
 
   @override
-  bool support(dynamic arg) => arg != null && arg is Map;
+  bool support(dynamic arg) => arg != null && arg is Map<String, dynamic>;
 
   @override
   ProcessedArg process(
     JsonWidgetRegistry registry,
     dynamic arg,
     Set<String>? jsonWidgetListenVariables,
   ) {
-    final mapArg = arg as Map;
+    final mapArg = arg as Map<String, dynamic>;
     final calculateListenVariables = jsonWidgetListenVariables == null;
     final resultListenVariables = jsonWidgetListenVariables ?? <String>{};
-    final processedMapArg = {};
+    final processedMapArg = <String, dynamic>{};
```

Comment:
Does the processMapArg need support other types than String as key? If so we could make this parser generic.
String is required by serialization parsing (as json only supports String as key).